libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -74,6 +79,10 @@ class AP_Scheduler
         uint16_t max_time_micros;
     };
 
+    enum class Options : uint8_t {
+        RECORD_TASK_INFO = 1 << 0
```

Comment:
I think it's good to set the number 1.

Reply:
This is consistent with bitmask values elsewhere - it makes is consistent with having 1 << 1, 1 << 2 etc